generate_manifest

```diff
@@ -52,6 +60,13 @@ fi
 APP_ID=$1
 FLATPAK_BRANCH=${FLATPAK_BRANCH:-master}
 GIT_BRANCH=${GIT_BRANCH:-master}
+GIT_PATH=${GIT_PATH:-.}
+GIT_COMMIT=""
+if [ -n "$FLATHUB" ]; then
+  # If flathub we get the branch commit sha to add to the manifest
+  COMMIT=$(wget -O - https://api.github.com/repos/endlessm/hack-toy-apps/git/ref/heads/${GIT_BRANCH} 2> /dev/null | grep sha | cut -d '"' -f4)
```

Comment:
Have you considered getting the SHA with git directly?
```
$ git fetch && git rev-parse origin/master
6c6c141aa96a469928da3b393f454ff3abf081af
```
I'm not sure about introducing a github specific api call. Which one you think is better?

Reply:
Yes, with git should be easier, I'll change that. It's just I didn't find the correct way to do that and sometimes I don't see the simplest solution and try to do something more complex. But your solution is a lot simpler.